docs/KLL/UnderstandingKLLBounds.md

```diff
@@ -63,30 +63,20 @@ public class QuantileBoundsTest {
     println("r2: " + r2);
     println("");
     
-    double r1LB = sk.getRankLowerBound(r1);
-    println("r1LB: " + r1LB);
-    double r1UB = sk.getRankUpperBound(r1);
-    println("r1UB: " + r1UB);
-    
-    double r2LB = sk.getRankLowerBound(r2);
-    println("r2LB: " + r2LB);
-    double r2UB = sk.getRankUpperBound(r2);
-    println("r2UB: " + r2UB);
+    println("r1LB(r1): " + sk.getRankLowerBound(r1));
+    println("r1UB(r1): " + sk.getRankUpperBound(r1));
+    println("r2LB(r2): " + sk.getRankLowerBound(r2));
+    println("r2UB(r2): " + sk.getRankUpperBound(r2));
     println("");
     
-    double q1LB = sk.getQuantileLowerBound(r1);
-    println("q1LB(r1): " + q1LB);
-    double q1UB = sk.getQuantileUpperBound(r1);
-    println("q1UB(r1): " + q1UB);
-    
-    double q2LB = sk.getQuantileLowerBound(r2);
-    println("q2LB(r2): " + q2LB);
-    double q2UB = sk.getQuantileUpperBound(r2);
-    println("q2UB(r2): " + q2UB);
+    println("q1LB(r1): " + sk.getQuantileLowerBound(r1));
+    println("q1UB(r1): " + sk.getQuantileUpperBound(r1));
+    println("q2LB(r2): " + sk.getQuantileLowerBound(r2));
+    println("q2UB(r2): " + sk.getQuantileUpperBound(r2));
     println("");
   }
   
-  static void println(Object o) { System.out.println(o.toString()); }
+  private static void println(Object o) { System.out.println(o.toString()); }
```

Comment:
avoiding things like this, which detract from the main focus of the example, is why i increasingly prefer python for these sorts of things.